src/main/java/com/kodedu/boot/AppStarter.java

```diff
@@ -271,6 +278,17 @@ private void registerStartupListener(CmdlineConfig config) {
                 });
             });
         }
+        if (config.workingDirectory != null) {
```

Comment:
It looks nice thank you, I have just one question, it seems to me it can be better to set working directory before adding config.files to tab. Can we move line 281 before line 267 ?

Reply:
Thought the order was not relevant as everything is threaded. Moving it